edx_user_state_client/interface.py

```diff
@@ -34,6 +34,19 @@ def __repr__(self):
         )
 
 
+class XBlockFieldModDate(namedtuple('_XBlockFieldModTime', ['block_key', 'field', 'updated'])):
+    """
+    Modification time for an XBlock field.
+
+    Arguments:
+        block_key: The :class:`UsageKey` of the XBlock.
+        field: The name of the modified field.
+        updated: A :class:`datetime.datetime` that identifies when this field
+                 was modified.
```

Comment:
Annoyingly, now that I look at it, we have an `updated` field on `XBlockUserState` which covers all of the fields together. The split of semantics between that and `get_mod_date` seems a bit strange. Thoughts? Is it worth keeping them separate to make sure that we have per-field modification times? Should `updated` in `XBlockUserState` be formalized to be the latest modification time of any of the states listed in the `state` dict?